starter_code/app.js

```diff
@@ -1,21 +1,48 @@
 
 const express = require('express');
-const hbs     = require('hbs');
+const ejs     = require('ejs');
 const app     = express();
 const path    = require('path');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const punkAPI = new PunkAPIWrapper();
+const expressLayouts = require('express-ejs-layouts');
 
-app.set('view engine', 'hbs');
+app.use(expressLayouts);
+app.set('layout', 'layout');
+app.set('view engine', 'ejs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static('public/stylesheets'));
```

Comment:
I'm not sure you need these two lines of code. With the line 14 should be enough.